src/AutoRest.CSharp/LowLevel/Output/LowLevelClient.cs

```diff
@@ -134,6 +139,45 @@ public static IEnumerable<LowLevelClientMethod> BuildMethods(RestClientBuilder b
             }
         }
 
+        private LowLevelConvenienceMethod BuildConvenienceMethod(in LowLevelClientMethod clientMethod, RestClientBuilder builder)
+        {
+            List<Parameter> parameters = new List<Parameter>();
+            MethodSignature protocolSignature = clientMethod.Signature;
+            InputOperation operation = clientMethod.RequestMethod.Operation;
+            Parameter? bodyParameter = null;
+            foreach (var parameter in protocolSignature.Parameters)
+            {
+                if (parameter == KnownParameters.RequestContent || parameter == KnownParameters.RequestContentNullable)
+                {
+                    bodyParameter = builder.GetBodyParameter(operation, DefaultNamespace);
+                    parameters.Add(bodyParameter!);
+                }
+                else if (parameter == KnownParameters.RequestContext)
+                {
+                    parameters.Add(KnownParameters.CancellationTokenParameter);
+                }
+                else
+                {
+                    parameters.Add(parameter);
+                }
+            }
+
+            CSharpType? responseType = builder.GetReturnType(operation, DefaultNamespace);
+            CSharpType? returnType = responseType == null ? typeof(Azure.Response) : new CSharpType(typeof(Azure.Response<>), responseType!);
```

Comment:
Can we fold this line into `GetReturnType()`?

Reply:
I'm afraid not. I need two things: `Response<responseType>` and `responseType`. `Response<responseType>` will pass into the `MethodSignature` to generate signature, and `responseType` will pass into `LowLevelConvenienceMethod` for model deserialization.

Reply:
Then I suggest to rename this method to `GetResponseType`, since `RestClientBuilder` is low level.